New recipe: 750words

zzamboni opened the pull request:
### Brief summary of what the package does

A library of posting text from Emacs to 750words.com.

### Direct link to the package repository

https://github.com/zzamboni/750words-client/

### Your association with the package

I am the author.

### Relevant communications with the upstream package maintainer

None needed.

### Checklist

<!-- Please confirm with `x`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I have confirmed some of these without doing them

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.

> Also includes an Org exporter

I think in this case you would want two separate packages, one with a dependency on the other -- that way each package can have its own prefix, and you can avoid:

```
45:0: error: "org-750words-export-to-750words" doesn't start with package's prefix "750words".
```

Also worth noting  on 750words.el#L28: Use empty lines instead of `;;` on a line by itself within your Commentary

zzamboni commented:
@riscy thanks for the feedback! I have split the Org exporter into a separate library: https://github.com/melpa/melpa/pull/7635.

I've also addressed the empty-lines in commentary.

riscy commented:
Looks great, thank you!
